packages/home-spa/.env.example

```diff
@@ -6,3 +6,4 @@ OP_SUBSCRIPTIONS_URL=
 
 ## SENTRY DSN
```

Comment:
Can you please remoce the duplicate env variable and comment

Reply:
both are required